utility/src/main/java/org/oppia/android/util/parser/html/HtmlParser.kt

```diff
@@ -80,6 +90,10 @@ class HtmlParser private constructor(
     return ensureNonEmpty(trimSpannable(spannableBuilder))
   }
 
+  private fun isRtlLayout(view: View): Boolean {
+    return view.layoutDirection == View.LAYOUT_DIRECTION_RTL
```

Comment:
Similar to your other PR, shouldn't we be using ViewCompat for checking the layout direction?

Reply:
Yes, right. Thanks fixed it.